linalg/vector.hpp

```diff
@@ -159,6 +159,11 @@ class Vector
    /// Reset the Vector to use the given external Memory @a mem and size @a s.
    /** If @a own_mem is false, the Vector will not own any of the pointers of
        @a mem.
+
+       Note that when @a own_mem is true, the @a mem object can be destroyed
+       immediately by the caller but `mem.Delete()` should NOT be called since
```

Comment:
Can be destroyed but not deleted?

Reply:
Destroying the `mem` object (invoking its destructor) does not automatically invoke `mem.Delete`. How should I update the comment to clarify this?